### DISCUSSION TITLE
Open multi instances when I double click toolbar icon on windows 10

### DISCUSSION THREAD

**Discussion by poolqa** [rustdesk/rustdesk#7143] [General]
### Bug Description

Open multi instance windows.

### How to Reproduce

just double click toolbar icon on windows 10

### Expected Behavior

open one instance windows.

### Operating system(s) on local side and remote side

windows 10

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/11583307/db6c4efa-9426-454c-9a2a-28e78da01445)

**Comment by rustdesk** [comment]
Thanks, https://github.com/rustdesk/rustdesk/commit/5fb026b8d5a9cc758e86c659f545196c262fd912
